README.md

```diff
@@ -7,7 +7,7 @@ This repository contains the https://satellytes.com website.
 To run the project all you need is:
 
 - Yarn
-- Node >= 16
+- Node >= 18
```

Comment:
The project needs >= 18 for quite some time now. I took the opportunity to adjust the readme.